http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeServiceInterpSpec.scala

```diff
@@ -819,34 +821,65 @@ class RuntimeServiceInterpTest
   }
 
   it should "get a runtime" in isolatedDbTest {
+    val mockSamService: SamService[IO] = mock[SamService[IO]](defaultMockitoAnswer[IO])
```

Comment:
This should be the default response of `MockSamService`, so not sure you need to override the mock here.

Reply:
MockSamService isn't a Mockito mock, though, and I wanted to have access to the method stubbing/mocking for my tests.